INFO     root:data_depend.py:37 提取变量：erroranswer: 582ca170-6860-4b93-afe4-50b9f5b9925e
INFO     root:validate.py:76 响应断言:  预期值: '200' 实际值: '200', 断言成功!